lib/stripe/webhook.rb

```diff
@@ -108,8 +108,9 @@ def self.verify_header(payload, header, secret, tolerance: nil)
         end
 
         if tolerance && timestamp < Time.now - tolerance
+          formatted_timestamp = Time.at(timestamp).strftime('%F %T')
```

Comment:
Sorry for the rubocop hassle, looks like these need to be double quotes. 😓

Reply:
Oh weird, that's what I had originally but rubocop corrected it 🤔 

```
lib/stripe/webhook.rb:112:82: C: [Corrected] Style/StringLiteralsInInterpolation: Prefer single-quoted strings inside interpolations.
```

But running rubocop locally again, this time it changes it back to double quotes:
```
lib/stripe/webhook.rb:111:61: C: [Corrected] Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```